### PR TITLE
Add a handy security radio

### PR DESCRIPTION
**Pull request by Ramlik**
## About the PR
I've always wanted to have a handy security radio in game and I thought it would be cool to have one in game.

## Why / Balance
This could help security guards in game for communication and give a good opportunity to all humanoid antagonists to listen all the talks of security through it. 

## Technical details
This walkie talkie works the same way as handheld one, but this one has recolored sprite and uses security channel.

## Media
https://youtu.be/VQOLiTQAmKc?si=IoeRN4mHT9vDIcIG

- [X] I have added screenshots/videos to this PR showcasing its changes ingame

## Breaking changes
Unless the prototype and/or sprites won't be loaded in game everything should work fine. Everything is tested on local server.

:cl:
- add: Added security walkie talkies to the vendor.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Devices/securityradio.yml
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Devices/securityradio.yml:1]: dont create a new file, just put that radio next to the existing one

**Review by lzk228 — CHANGES_REQUESTED** [review]
more requests
- use default radio as a parent
- make a new .rsi file and make the same png names in both files, so you shouldnt change them in sec radio (theyll inherit from parent)

**Comment by Brandon-Huu** [comment]
So just a more annoying chat that can't be sabotaged?

**Comment by Hanzdegloker** [comment]
> So just a more annoying chat that can't be sabotaged?

I mean, yeah

**Comment by UbaserB** [comment]
i feel like the sprite needs improving, its too shiny

**Comment by Krunklehorn** [comment]
Why is this security specific? Why not just change handheld radios to take encryption keys?

**Comment by IProduceWidgets** [comment]
I think it would be better to make handheld radios have changeable frequencies, but iirc there was a PR for that which got mothballed.

File: Resources/Prototypes/Catalog/Fills/Lockers/security.yml [hunk 1]
```diff
@@ -52,6 +52,8 @@
   components:
   - type: StorageFill
     contents:
+      - id: RadioHandheldSecurity
+        prob: 0.5
       - id: FlashlightSeclite
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/Fills/Lockers/security.yml:57]: Put it in the vendor it's not essential for sec role.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**